### PR TITLE
feat(rabbitmq): add rabbitmq

### PR DESCRIPTION
**Pull request by bacongobbler**

### CODE REVIEW

File: rabbitmq/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Alpine
```
Comment (technosophos) [inline rabbitmq/README.md]: Probably change title to RabbitMQ

Comment (bacongobbler) [inline rabbitmq/README.md]: oops, indeed

File: rabbitmq/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+This Chart provides a Replication Controller running RabbitMQ.
```
Comment (bacongobbler) [inline rabbitmq/README.md]: note to self: this should also get changed (switched to a pod)

File: rabbitmq/README.md (hunk 1 above)
Comment (mboersma) [inline rabbitmq/README.md]: README.md still needs to update this headline.

**Pull request merged**